app/views/content/life-as-a-teacher/explore-subjects.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "Explore subjects you could teach"
+heading: "Explore subjects"
+category: Life as a teacher
+image: "static/images/content/hero-images/physics_1383.jpg"
+title_paragraph: Discover what you could teach, the benefits of teaching specific subjects and how to take your next steps to become a teacher. 
+description: |-
+  Discover what you could teach, the benefits of teaching specific subjects and how to take your next steps to become a teacher.
```

Comment:
@emmaachesongray Do you want to update this as well to reflect the new description? This is the meta description so I think your new wording probably makes more sense